src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = "///";
+    private static final int INDEX_0 = 0;
+    private static final int INDEX_1 = 1;
```

Comment:
constant names should represent the conceptual point of view and what does it mean for us and not just a fancy number name
If you want to reuse some constant value with different ideas, it is better to create another one with the same value and name it right
```suggestion
    private static final int OPERATION_INDEX = 0;
    private static final int AMOUNT_INDEX = 1;
```